src/main/java/unicash/logic/commands/GetTotalExpenditureCommand.java

```diff
@@ -21,17 +17,9 @@
  * Calculates and returns the total expenditure of a user in a given month and (optionally) category and year.
  */
 public class GetTotalExpenditureCommand extends Command {
-    public static final String COMMAND_WORD = "get_total_expenditure";
-
-    public static final String MESSAGE_USAGE = new CommandUsage.Builder()
-            .setCommandWord(COMMAND_WORD)
-            .setDescription("Retrieves the total expenditure by month with optional filters for category and year.")
-            .addParameter(PREFIX_MONTH, "Month")
-            .addParameter(PREFIX_CATEGORY, "Category", true, false)
-            .addParameter(PREFIX_YEAR, "Year", true, false)
-            .setExample(ExampleGenerator.generate(COMMAND_WORD, PREFIX_MONTH, PREFIX_CATEGORY, PREFIX_YEAR))
-            .build()
-            .toString();
+    public static final String COMMAND_WORD = CommandType.GET_TOTAL_EXPENDITURE.getCommandWords();
+
+    public static final String MESSAGE_USAGE = CommandType.GET_TOTAL_EXPENDITURE.getMessageUsage();
 
     public static final String MESSAGE_SUCCESS = "Your total expenditure in %1$s %2$d was %3$.2f";
```

Comment:
Is there no message success here? Because I am adding a slight variation to the success message when they have a category so I can display 2 types of messages

Reply:
oh you are right, I missed this out! thanks for pointing it out